MAUI/DataForm/getting-started.md

```diff
@@ -165,6 +165,78 @@ this.Content = dataForm;
 
 N> [View sample in GitHub](https://github.com/SyncfusionExamples/maui-dataform/tree/master/GettingStarted)
 
+## DataForm inside stack layout
+
+#### Vertical StackLayout
+
+When the data form is placed inside a vertical stack layout, users must define the required `MinimumHeightRequest` value. By default, this `MinimumHeightRequest` is set to 300.
```

Comment:
When the data form is placed inside a vertical stack layout, you must define the required `MinimumHeightRequest` value. By default, this `MinimumHeightRequest` is set to 300.

Reply:
Changed the content